subprojects/docs/src/docs/userguide/authoring-builds/gradle_init.adoc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2023 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[gradle_init]]
+= Gradle Init
+
+For users that are starting a new project, Gradle comes with a built-in task called `init` that initializes a Gradle project.
+
+This ensures that you have the minimum set of requirements for Gradle to work when developing a new app, plugin, or library.
+
+The `init` task uses the built-in `wrapper` task to create a Gradle wrapper script, `gradlew` and `gradle.bat`.
+Then, based on a set of user inputs, the task creates default Gradle files and directories.
+
+== The init task
+
+The `init` task must be run in an empty folder and Gradle must already be installed.
+
+Run the `init` task using the following command in a terminal:
+
+----
+$ gradle init
+----
+
+== Task options
+
+The task prompts the user for information about the new project:
+
+----
+$ gradle init
+
+Select type of project to generate:
+  1: basic
+  2: application
+  3: library
+  4: Gradle plugin
+Enter selection (default: basic) [1..4]
+
+Select build script DSL:
+  1: Groovy
+  2: Kotlin
+Enter selection (default: Kotlin) [1..2]
+
+Project name (default: project):
+
+Generate build using new APIs and behavior (some features may change in the next minor release)? (default: no) [yes, no]
+----
+
+There are 4 types of project:
+
+1. *Basic* - a basic project with one settings file and one build file.
+2. *Application* - a C++, Groovy, Java, Kotlin, Scala or Swift project with an `app` folder.
+3. *Library* - a library such as an API with a `lib` folder.
+4. *Gradle Plugin* - a Gradle plugin with a `plugin` folder.
+
+You will be asked if your scripts are written in Groovy or Kotlin.
+Kotlin is the recommended default.
+Your choice determines which DSL you will use to configure Gradle and this can be changed at any time.
```

Comment:
Hmmmm I'm not sure about the wording. Sure it can be changed later - but it could be a lot of work. This makes it sound like "no worries - you can change this easily later"